orchestrator/taste-orchestrator.py

```diff
@@ -820,6 +823,54 @@ def BuildGUIs(guiSubsystems, cflagsSoFar, asn1Grammar):
         os.chdir("../..")
 
 
+def BuildRosBridges(rosBridgeSubsystems, cflagsSoFar, asn1Grammar, acnFile):
+    '''Builds automatically generated ROS bridges'''
+    if rosBridgeSubsystems:
+        g_stageLog.info("Building automatically created ROS bridges")
+    for baseDir in rosBridgeSubsystems:
+        olddir = os.getcwd()
+        if not os.path.isdir(baseDir):
+            panic("No directory %s! (pwd=%s)" % (baseDir, os.getcwd()))
```

Comment:
I thought `"".format()` was recommended.
But it would be ok to keep it as is if this is the way formatting is done throughout the code base.

Reply:
Yes it is allover the place.